### PR TITLE
Add recipe for echo

### PR DESCRIPTION
**Pull request by astrofrog**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] <s>If static libraries are linked in, the license of the static library is packaged.</s>
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by astrofrog-conda-forge** [comment]
I am happy to be listed as a maintainer! (from this account)

File: recipes/echo/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    - setuptools_scm
+  run:
+    - python
+    - numpy
+    - qtpy
+    - setuptools
+
+test:
+  imports:
+    - echo
```
Comment (synapticarbors) [inline recipes/echo/meta.yaml:30]: it would probably be good to add to the imports tests `- echo.qt`

Comment (astrofrog) [reply recipes/echo/meta.yaml:31]: Done

File: recipes/echo/LICENSE [hunk 2]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```
[the remark this answers is not in the document]
Comment (astrofrog) [reply recipes/echo/LICENSE]: Done

**Comment by astrofrog** [comment]
@synapticarbors - thanks for the review! The CI is passing now.

**Comment by astrofrog** [comment]
@synapticarbors - the generated repository is empty, what could be happening? (https://github.com/conda-forge/echo-feedstock)

**Comment by synapticarbors** [comment]
This appears to be a system wide issue:

https://github.com/conda-forge/status/issues/80

It should hopefully be resolved soon and then will fix the feedstock automatically.

**Pull request merged**